config/rubocop/config/rspec.yml

```diff
@@ -42,6 +42,7 @@ RSpec/FilePath:
     - "spec/lib/dfe/wizard_store_spec.rb"
     - "spec/lib/dfe/wizard_spec.rb"
     - "spec/lib/dfe/wizard_step_spec.rb"
+    - "spec/lib/dfe/bigquery/*_spec.rb"
```

Comment:
You should be able to replace these exceptions with

```
RSpec/FilePath:
  CustomTransform:
    DfE: dfe
    DfESignInButtonComponent: dfe_sign_in_button_component
```

Reply:
Good one! Done!